python/py-cryptography/Portfile

```diff
@@ -32,18 +33,93 @@ platform macosx {
         patchfiles-append   patch-src__cffi_src_build_openssl.py.diff
     }
 }
+if {${name} ne ${subport}
+    && [string match "py*-${github.project}" ${subport}]} {
+    depends_build-append \
+                    port:py${python.version}-setuptools-rust \
+                    port:rust
+
+    depends_lib-append \
+                    path:lib/libssl.dylib:openssl \
+                    port:py${python.version}-asn1crypto \
```

Comment:
also since version 2.8 it seems that `asn1crypto` is no longer a [dependency](https://github.com/pyca/cryptography/blob/b2f9a0e1a546c3b10a6e96cd5e8423a7b04970c7/CHANGELOG.rst#28---2019-10-16).

Reply:
Thanks fixed.